lib/api/socket-handler.js

```diff
@@ -31,6 +32,15 @@ var SocketHandler = function(app, server, auth) {
   var self = this;
   this.io.sockets.on('connection', function(socket) {
     var clientQuery;
+    
+    // override the emit function to add logging
+    var emit = socket.emit;
+    socket.emit = function(event, data) {
+      logger.debug(event, data);
+      
+      socket.emit = emit;
+      socket.emit(event, data);
```

Comment:
Could 41-42 just be `emit(event, data)`?

Why are you reassigning socket.emit on 41? Wouldn't that remove the overridden handler?